crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -83,14 +83,15 @@ impl SystemConfigs {
         })
     }
 
-    pub(crate) fn in_set_inner(&mut self, set: BoxedSystemSet) {
+    /// Adds a new boxed system set to the systems.
+    pub fn in_set_dyn(&mut self, set: BoxedSystemSet) {
```

Comment:
Note that this change will be unnecessary if #7762 is merged.

Reply:
Should I revert that change?

Reply:
Nah, that PR isn't guaranteed to get merged. Just thought I'd mention that this function might get removed later.

Reply:
Oh, okay